build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -98,7 +104,10 @@
 
     <!-- Workaround for https://github.com/dotnet/sdk/issues/115 -->
     <ItemGroup>
-      <HackFilesToCopy Include="$(NuGetPackagesDir)\microsoft.build.runtime\$(CLI_MSBuild_Version)\contentFiles\any\netcoreapp1.0\**;$(NuGetPackagesDir)\microsoft.codeanalysis.build.tasks\2.0.0-beta6-60922-08\contentFiles\any\any\**;$(NuGetPackagesDir)\microsoft.testplatform.cli\$(CLI_TestPlatform_Version)\contentFiles\any\any\**" />
+      <HackFilesToCopy Include="$(NuGetPackagesDir)\microsoft.build.runtime\$(CLI_MSBuild_Version)\contentFiles\any\netcoreapp1.0\**;
```

Comment:
Yeah, I'm aware of it, but I still saw failures when I tried removing it. In the interest of keeping an already-oversized changeset smaller, I'm postponing digging into this until after we get to green on the netcoreapp1.1 change.